src/frame/js/web-messenger.jsx

```diff
@@ -194,13 +194,13 @@ export function init(props = {}) {
                                     storage.removeItem(`${props.appId}.appUserId`);
                                     storage.removeItem(`${props.appId}.sessionToken`);
 
-                                    return store.dispatch(batchActions([
+                                    store.dispatch(batchActions([
```

Comment:
I think it was just accidentally changed here https://github.com/smooch/smooch-js/pull/595/commits/d87409c5ca6980c97a82785c1aec200b849b839a#diff-980e1568ab82d9ffbf8e1c57ff157d00L206

/shrug